homeassistant/components/onewire/sensor.py

```diff
@@ -129,6 +258,22 @@ def unit_of_measurement(self):
         return self._unit_of_measurement
 
 
+class OneWireProxy(OneWire):
+    """Implementation of a One wire Sensor through owserver."""
+
+    def update(self):
+        """Get the latest data from the device."""
+        value = None
+        try:
+            value_read = self._read_value_ownet()
+            if len(value_read) > 0:
+                value = round(float(value_read), 1)
```

Comment:
Please move this out of the try... except block. We only want to wrap the line(s) that can raise. If needed we can use:
```py
try:
    ...
except:
    ...
else:
    ...
```